src/pages/stacks-blockchain/sending-tokens.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Sending tokens
+description: Learn how to transfer tokens
+icon: TestnetIcon
+duration: 15 minutes
+experience: beginners
+tags:
+  - tutorial
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Introduction
+
+This tutorial will walk you through the following steps:
+
+- Specifying a sender key
+- Generating a token transfer transaction
+- Broadcasting the transaction to the network
+- Checking transaction completion
+- Confirming updates account balances (optional)
+
+## Requirements
+
+You will need [NodeJS](https://nodejs.org/en/download/) `8.12.0` or higher to complete this tutorial. You can verify your installation by opening up your terminal and run the following command:
+
+```bash
+node --version
+```
+
+You should also complete the [Managing accounts tutorial](/stacks-blockchain/managing-accounts). The following steps assume we have access to an exsiting Stacks 2.0 account.
+
+## Step 1: Installing libraries
+
+First install the stacks transactions library, a JavaScript [BigNum](https://en.wikipedia.org/wiki/Arbitrary-precision_arithmetic) implementation, and the API client for the [Stacks 2.0 Blockchain API](/references/stacks-blockchain):
+
+```shell
+npm install --save @blockstack/stacks-transactions@0.6.0 bn.js @stacks/blockchain-api-client
+```
+
+## Step 2: Specifying a sender
+
+In order to build and sign transactions, you will need a Stacks privat key. You can easily generate a new, random Stacks 2.0 sender key (see ["Generating an account" from the previous tutorial](http://localhost:3000/stacks-blockchain/managing-accounts#step-2-generating-an-account)).
+
+For this tutorial, we will use an existing Stacks account and instantiate the key object from a private key string:
+
+```js
+const BN = require('bn.js');
+const {
+  makeSTXTokenTransfer,
+  createStacksPrivateKey,
+  broadcastTransaction,
+  estimateTransfer,
+  getNonce,
+  StacksTestnet,
+  privateKeyToString,
+} = require('@blockstack/stacks-transactions');
+const { TransactionsApi, Configuration } = require('@stacks/blockchain-api-client');
+
+const apiConfig = new Configuration({
+  fetchApi: fetch,
+  basePath: 'https://stacks-node-api-latest.argon.blockstack.xyz',
+});
+
+const key = 'edf9aee84d9b7abc145504dde6726c64f369d37ee34ded868fabd876c26570bc01';
+const senderKey = createStacksPrivateKey(key);
+```
+
+-> Note: The code above also imports methods required for the next steps, including API configuration for the client library usage.
+
+## Step 3: Generating transaction
+
+To generate a token transfer transaction, we will be using the `makeSTXTokenTransfer()` transaction builder function:
+
+```js
+const recipient = 'SP3FGQ8Z7JY9BWYZ5WM53E0M9NK7WHJF0691NZ159';
+
+// amount of Stacks tokens to send (in microstacks). 1,000,000 microstacks are worth 1 STX token
+const amount = new BN(1000000);
+
+// skip automatic fee estimation
```

Comment:
I found this confusing. If we're skipping automatic fee estimation, shouldn't we be setting some meaningful values for the fee (or is 0 a useful default)?

Reply:
good point!